src/components/core/DpEditor/DpEditor.stories.mdx

```diff
@@ -16,4 +16,7 @@ A rich text editor component based on tiptap.
 - the editor-content component needs a prop with editor instance to work correctly
```

Comment:
These content above is tailored to the specific implementation within demosplan. I'd suggest to delete all of it.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/567/commits/df50c169fd1b50ab220d9f778e973311f2f03123